TAO/tests/Oneway_Send_Timeouts/Server_Task.h

```diff
@@ -47,7 +53,8 @@ class Server_Task : public ACE_Task_Base
 
   bool ready ()
   {
-    if (server_.get() != 0) {
+    if (server_.get () != 0)
```

Comment:
get() can be removed here, we can use operator bool now, more places in this file

Reply:
done